x/auction/types/auctions.go

```diff
@@ -66,15 +68,35 @@ func (a BaseAuction) GetType() string { return "base" }
 
 // Validate verifies that the auction end time is before max end time
 func (a BaseAuction) Validate() error {
-	if a.EndTime.After(a.MaxEndTime) {
-		return fmt.Errorf("MaxEndTime < EndTime (%s < %s)", a.MaxEndTime, a.EndTime)
+	// ID can be 0 for surplus, Debt and collateral auctions
```

Comment:
ID is not used for any but the base auction type. Should we consider removing it? @karzak

Reply:
the ID will always be 0 for the named actions though

Reply:
Yes, the ID is set by the keeper when it is first stored.